Milestone1: EDR Added

arslanbisharat opened the pull request:
Worked on these features:
>Create your own ERD. You can use an online tool to do that or draw it on the piece of paper and take a photo of it.
>Think about friendship and friendship requests. Small hint: try to use status of the friendship to tell if it is a request or confirmed friendship.
>Extra hint: keep profile image as a link to gravatar as storing files is a bit tricky in Heroku.
>Put a file with your ERD in the docs directory in your repo.

** NOTE: I Worked Alone on This Project

uche-inyama reviewed (approved):
## Approved

Hi arslan :wave: 

This project is approved. :100: 

I can see on your pull request, you are worked alone on this milestone.

However, if you are going to work alone on this project, please inform  student success team - @Willow. :slightly_smiling_face: 


#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

arslanbisharat committed:
Milestone1: EDR Added

arslanbisharat committed:
Gem updated

arslanbisharat committed:
generate friendrequests model and create model associations ang window gem

arslanbisharat committed:
update friendrequests  model

arslanbisharat committed:
Adds the following

- Installs gem files
- Install rspec and capybara
- adds test for user model

arslanbisharat committed:
adds test for post and friend request models

arslanbisharat committed:
Update tests for post model and add tests for like model

arslanbisharat committed:
Add tests for the comment model Add integration tests for authenticating a user

arslanbisharat committed:
Update factories folder

arslanbisharat committed:
Add integration tests creating a new post and some other setting

arslanbisharat committed:
Add itegration tests for comment and like models

arslanbisharat committed:
linters

arslanbisharat committed:
controller settings

arslanbisharat committed:
Add the following:

- Update the user controller and model
- Update the friend request controller
- Update index and show pages
- Add friends link to the nav bar

arslanbisharat committed:
Add rspec tests for the controllers

arslanbisharat committed:
adds feature test for friend request

arslanbisharat committed:
Add tests to helper methods for the user model

arslanbisharat committed:
linter errors

arslanbisharat committed:
current user reomved

arslanbisharat committed:
linter
